client/pkg/omni/resources/omni/extensions_configuration.go

```diff
@@ -14,7 +14,7 @@ import (
 	"github.com/siderolabs/omni/client/pkg/omni/resources"
 )
 
-// NewExtensionsConfiguration creates new extensions configuration resource.
+// NewExtensionsConfiguration creates new Extensions configuration resource.
```

Comment:
Maybe if we change that it should be?
```suggestion
// NewExtensionsConfiguration creates new ExtensionsConfiguration resource.
```